File: 시즌 11/중/kimryungyo/other_languages/day_1.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        int majors = scanner.nextInt();
        int total = scanner.nextInt();
        int remain_semester = 8 - N;

        for (int i = 0; i < remain_semester; i++) {
            int major = scanner.nextInt();
            int non_major = scanner.nextInt();

            for (int j = 0; j < 6; j++) {
                if (major > 0) {
                    majors += 3;
                    total += 3;
                    major -= 1;
                } else if (non_major > 0) {
                    total += 3;
                    non_major -= 1;
                }
            }
        }

        if (total >= 130 && majors >= 66) {
            System.out.println("Nice");
        } else {
            System.out.println("Nae ga wae");
        }
    }
}

File: 시즌 11/중/kimryungyo/other_languages/day_1.c
#include <stdio.h>
int main(void) {
	int N, A, B;
	int major, non_major;

	scanf("%d %d %d", &N, &A, &B);

	for (int i = 0; i < (8-N); i++) {
		scanf("%d %d", &major, &non_major);
		for (int j = 0; j < 6; j++) {
			if (major > 0) {
				A += 3;
				B += 3;
				major -= 1;
			} else if (non_major > 0) {
				B += 3;
				non_major -= 1;
			}
		}
	}

	if ((A >= 66) && (B >= 130)) {
		printf("Nice\n");
	} else {
		printf("Nae ga wae\n");
	}

}